multiwallet.go

```diff
@@ -6,46 +6,42 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
-	"strconv"
 	"strings"
-	"sync"
-	"time"
 
 	"decred.org/dcrwallet/v2/errors"
-	w "decred.org/dcrwallet/v2/wallet"
 	"github.com/asdine/storm"
 	"github.com/asdine/storm/q"
 	"github.com/decred/dcrd/chaincfg/v3"
 	"github.com/planetdecred/dcrlibwallet/utils"
-	"github.com/planetdecred/dcrlibwallet/walletdata"
-	bolt "go.etcd.io/bbolt"
+
+	"github.com/planetdecred/dcrlibwallet/wallets/dcr"
+
 	"golang.org/x/crypto/bcrypt"
 )
 
-type MultiWallet struct {
-	dbDriver string
-	rootDir  string
-	db       *storm.DB
-
-	chainParams *chaincfg.Params
-	wallets     map[int]*Wallet
-	badWallets  map[int]*Wallet
-	syncData    *syncData
+type Assets struct {
+	DCR struct {
+		Wallets     map[int]*dcr.Wallet
+		BadWallets  map[int]*dcr.Wallet
+		DBDriver    string
+		RootDir     string
+		DB          *storm.DB
+		ChainParams *chaincfg.Params
+	}
+}
 
-	notificationListenersMu         sync.RWMutex
-	txAndBlockNotificationListeners map[string]TxAndBlockNotificationListener
+type MultiWallet struct {
+	DbDriver string
+	RootDir  string
+	DB       *storm.DB
 
-	blocksRescanProgressListener     BlocksRescanProgressListener
-	accountMixerNotificationListener map[string]AccountMixerNotificationListener
+	ChainParams *chaincfg.Params
+	Assets      *Assets
```

Comment:
Do these properties NEED to be exported? They were private properties.

Reply:
![Screenshot from 2022-08-26 16-29-46](https://user-images.githubusercontent.com/25265396/186940269-89d2646c-4423-4b08-a3c9-50a53f519d20.png)

assets would need to be if we are going to access it's fields